lale/lib/sklearn/extra_trees_regressor.py

```diff
@@ -356,4 +356,29 @@
         set_as_available=True,
     )
 
+if sklearn.__version__ >= "1.1":
+    # old: https://scikit-learn.org/1.0/modules/generated/sklearn.tree.ExtraTreesRegressor.html
+    # new: https://scikit-learn.org/1.1/modules/generated/sklearn.tree.ExtraTreesRegressor.html
+    ExtraTreesRegressor = ExtraTreesRegressor.customize_schema(
+        max_features={
+            "anyOf": [
+                {"type": "integer", "forOptimizer": False},
```

Comment:
Should this be the same as the integer case in `ExtraTreesClassifier`?

Reply:
Or perhaps even shared as a common schema, so it can be reused for ExtraTreesClassifier, ExtraTreesRegressor, RandomForestClassifier, RandomForestRegressor.

Reply:
The Interger case is coming from the previous version of the operators (https://github.com/IBM/lale/blob/master/lale/lib/sklearn/extra_trees_regressor.py#L120 and https://github.com/IBM/lale/blob/master/lale/lib/sklearn/extra_trees_classifier.py#L118). Do you think that I should also refactor these?

Reply:
It's not really in the scope of this PR, so I'd only do it if it doesn't add too much work.